lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -952,6 +952,17 @@ def _create_deployment_parameters(
             parameters=parameters,
         )
 
+        # dump arm_template and arm_parameters to file
+        dump_path = os.path.join(os.getcwd(), "runtime/runs/arm_dump")
```

Comment:
1. LISA deployed several environments in a test run. So the files should put into folder of each environment.  call Environment.log_path to get the unique path.
2. LISA hides sensitive information in log files, so all dump content should pass through the secret method. call `secret.replace`.
3. LISA should avoid to put log file out of the folder of current run. use pathlib to detect and trim path, make sure the log put under current run. Try relative_path and join with env log path.

Reply:
2. Would secret.replace make the following string be output into the dump files? "******"
3. Could you please expand on the third point? I don't really understand what you want me to do for it.

Reply:
I hope to have a common method to finish the 2 and 3. It can be named `dump_file` and put in `utils/__init__.py`. 

> 2. Would secret.replace make the following string be output into the dump files? "******"
Yes, it's expected.

> 3. Could you please expand on the third point? I don't really understand what you want me to do for it.
For example, the file path may be "/etc/some_config" or "../../../../etc/some_config", it's invalid and dangerous to current system. So do some validation, 1) if it's an absolution path, make sure it's in scope of current run log path. 2) if it's a relative path, do same.